### PR TITLE
initial files added  

### PR DESCRIPTION
**Pull request by bashaerhaddad**
- [x] Added index file 

- [x] Added css file

### CODE REVIEW

**Review by Rahaf-96 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<html>
+  <head>
+    <link rel="stylesheet" ahref="style.css">
+  </head>
+  <body>
+
+  </body>
+  
+  <footer>
+  </footer>
```
Comment (Rahaf-96) [inline index.html]: You should put the footer inside the body tag

**Review by Rahaf-96 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" ahref="style.css">
```
Comment (saharAdem) [inline index.html:4]: Notice the spelling of `href` is wrong. remove 'a' letter from the beginning.

**Review by obaidofj — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  </head>
```
Comment (saharAdem) [inline index.html:5]: Also, add other meta tags to give your browser more information about your website. 
ex. 
`
  <meta charset="UTF-8">
`
`
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
`
` <title>Document</title>
`

**Comment by saharAdem** [comment]
Another time try to write a descriptive branch about your work. like you can name it  `file-structure` instead of your name.

**Pull request merged**